protected/views/patient/crud/_form.php

```diff
@@ -414,7 +414,7 @@
                     }'
                 ),
                 'htmlOptions' => array(
-                    'placeholder' => 'search Referred to',
+                    'placeholder' => 'search User',
```

Comment:
It is strange that this screen uses "search X", while other screens use "type to search for Xs"